system/datatypes/peer_reviewed_article.php

```diff
@@ -564,13 +567,16 @@ private function updateDiscussionsContainer($type, $discussionsContainer) {
 			$list = $this->xml->createElement('ul');
 			/** @var HyphaDomElement $reviewCommentContainer */
 			$reviewCommentContainer = $this->xml->createElement('div');
+			$reviewTitle = $this->getXml()->createelement('div',$_discussionnumberlabel);
+			$reviewTitle->attr('class','comment-subject');
 			$class = 'review-comment-wrapper collapsed';
 			foreach (['blocking' => $blocking, 'resolved' => $resolved, 'closed' => $closed] as $name => $isTrue) {
```

Comment:
As a result of this (?) the title is now shown as:

title:new title(nr. 2)

I would prefer to do it as follows:

#2 new title

So leave out the label ("title"), because it is obvious. Introduce the number at frond, because a number may also be part of the comment title itself.

Reply:
`<h3 class="comment-title"><span class="comment-number"># 3</span> <span class="title">Dit is de titel</span></h3>`